admin/templates/nodes/storage_status.html

```diff
@@ -1,11 +1,15 @@
-{% if not resource.is_public and resource.storage_limit_status is STORAGE_LIMITS.APPROACHING_PRIVATE.value %}
+{% if resource.storage_limit_status is STORAGE_LIMITS.NOT_CALCULATED.value %}
+    <span class="label label-warning">Storage limit not calculated, refresh or click Recalculate node storage usage button</span>
+{% elif not resource.is_public and resource.storage_limit_status is STORAGE_LIMITS.APPROACHING_PRIVATE.value %}
     <span class="label label-warning">Approaching storage limit for private nodes</span>
 {% elif resource.is_public and resource.storage_limit_status is STORAGE_LIMITS.APPROACHING_PUBLIC.value %}
     <span class="label label-warning">Approaching storage limit for public nodes</span>
 {% elif resource.is_public and resource.storage_limit_status is STORAGE_LIMITS.OVER_PUBLIC.value %}
     <span class="label label-danger">Node is over storage limit for public nodes</span>
 {% elif not resource.is_public and resource.storage_limit_status is STORAGE_LIMITS.OVER_PRIVATE.value %}
     <span class="label label-danger">Node is over storage limit for private nodes</span>
+{% elif not resource.is_public and not resource.storage_limit_status is STORAGE_LIMITS.DEFAULT.value %}
```

Comment:
Nit: I think the elif logic here would be a little easier to intuit if the `not resource.is_public` clauses were grouped together. 

Clarification: The reason we can't compare the storage_limit_status values using the integers (i.e. `resource.storage_limit_status > STORAGE_LIMITS.OVER_PRIVATE` is because the private storage cap limit may be higher than the public storage cap limit if we set custom storage caps? This may be a product question but I can't actually envision a situation in which we would want to grant custom private storage caps and not also adjust the public storage caps to be at least the same so that the private project could be made public.

Reply:
I agree that it would be silly to have a private cap be higher than a public cap, but if we're not enforcing it (which I don't think we should do). I'd rather be safer than sorry, if that makes sense.

Reply:
Complete sense! I was just pondering.